Jenkinsfile

```diff
@@ -164,7 +164,7 @@ pipeline {
                             filename 'Dockerfile.centos:7'
                             dir 'utils/docker'
                             label 'docker_runner'
-                            additionalBuildArgs "-t ${sanitized_JOB_NAME}-centos7 " + '$BUILDARGS'
+                            additionalBuildArgs "-t ${sanitized_JOB_NAME}-centos7 " + '$BUILDARGS --build-arg DONT_USE_RPMS=false --build-arg DONT_USE_RPMS=false OMPI=ompi-3.0.0rc4-1'
```

Comment:
Putting software versions inside a dockerfile isn't something that's currently done, would it be better to perform this action as part of the build step?